privacyidea/lib/config.py

```diff
@@ -997,13 +998,19 @@ def get_privacyidea_node(default='localnode'):
 def get_privacyidea_nodes():
     """
     This returns the list of the nodes, including the own local node name
+
     :return: list of nodes
+    :rtype: list
     """
-    own_node_name = get_privacyidea_node()
-    nodes = get_app_config_value("PI_NODES", [])[:]
-    if own_node_name not in nodes:
-        nodes.append(own_node_name)
-    return nodes
+    node_names = []
+    nodes = db.session.query(NodeName).all()
+    for node in nodes:
+        node_names.append(node.name)
+    # if there are no nodes in the database we add at least the local node to the list
```

Comment:
Should the app.py have written the local nodename to the database anyways?

Reply:
Yes, it should have. Done in c0c5f2d5e